app.js

```diff
@@ -0,0 +1,185 @@
+const monsterHealthBar = document.getElementById("monster-health");
+const playerHealthBar = document.getElementById("player-health");
+const bonusLifeEl = document.getElementById("bonus-life");
+
+const attackBtn = document.getElementById("attack-btn");
+const strongAttackBtn = document.getElementById("strong-attack-btn");
+const healBtn = document.getElementById("heal-btn");
+const logBtn = document.getElementById("log-btn");
+
+const enteredHealth = prompt("Choose maximum life for you and monster!", "100");
+
+let chosenMaxLife = parseInt(enteredHealth);
+
+if (isNaN(chosenMaxLife || chosenMaxLife <= 0)) {
```

Comment:
```suggestion
if (isNaN(chosenMaxLife) || chosenMaxLife <= 0) {
```
isNaN function must be closed before "||" this sign

Reply:
Thankyou for your suggestion. i am changing it @Tanvir04khan

Reply:
Thankyou for your suggestion. i made those changes and fixed the issue. can you please see the commits and review it @Tanvir04khan